dev-python/distro-oem/distro-oem-1.6.0.ebuild

```
@@ -0,0 +1,30 @@
+# Copyright 2021 Microsoft Corporation
+# Distributed under the terms of GNU General Public License v2
+
+EAPI=7
+
+MY_PN='distro'
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="OS platform information API"
+HOMEPAGE="https://github.com/python-distro/distro"
+SRC_URI="${HOMEPAGE}/releases/download/v${PV}/${MY_P}.tar.gz"
+
+LICENSE="Apache-2.0"
+KEYWORDS="amd64 arm64"
+
+SLOT="0"
+RDEPEND="dev-lang/python-oem"
```

```suggestion
RDEPEND="dev-lang/python-oem:3.6"
```
So that this fails at build time if someone forgets to update.

---

Updated it too with the `DEP_PYVAR` stuff.